app/views/documentation/webhooks.html.haml

```diff
@@ -0,0 +1,21 @@
+:markdown
+  # Webhooks
+
+  Webhooks allow you to run custom code when one of your scrapers has finished
```

Comment:
As someone who isn't very familiar with webhooks I find this explanation a bit confusing.

> Webhooks allow you to run custom code

When I read this I imagine that I can add some custom ruby code somewhere in morph that will run when my scraper finishes.

I think this would be clearer if it started with a basic explanation of why you might want a webhook and what it is. Something like (but simpler/shorter/clearer if possible :wink:):

> Ever wished you could be alerted whenever your morph.io scrapers finished running? Then you could send out notifications, update an archive, or run analysis as soon as new data is collected. This is what webhooks are for. You can add webhook urls to your scrapers and we'll make a HTTP POST request to it when your scraper finishes each run. You can then use this to trigger any custom code you like in an external application. Add webhook urls in your scrapers settings page.